README.md

```diff
@@ -8,6 +8,25 @@
 [![codecov](https://codecov.io/gh/jazzband/prettytable/branch/master/graph/badge.svg)](https://codecov.io/gh/jazzband/prettytable)
 [![Code style: Black](https://img.shields.io/badge/code%20style-black-000000.svg)](https://github.com/psf/black)
 
+PrettyTable's main goal is to let you print tables in an attractive ASCII form, like
+this:
```

Comment:
Thanks, this is a good idea, although it's a copy and paste of  https://github.com/jazzband/prettytable#displaying-your-table-in-ascii-form

Maybe simplify it a bit?

```suggestion
PrettyTable lets you print tables in an attractive ASCII form:
```